meson.build

```diff
@@ -1264,7 +1269,7 @@ endif
 # of meson newer than that.
 
 test_env = environment({'PICOLIBC_TEST' : '1'})
-test_env.prepend('PATH', meson.source_root() / 'scripts')
+test_env.prepend('PATH', meson.current_source_dir() / 'scripts')
```

Comment:
Any particular reason for this change? As this is run in the top-level meson.build file, wouldn't those be the same? Or is there some interaction with subprojects?

Reply:
source_root was deprecated in meson 0.56 but I've removed this change now we're not changing the meson version.

Reply:
yup, I reviewed the changes since 0.53 and found that too -- ce310201ab403f57874925e4165c66484ff0224d